### PR TITLE
chore: consolidate duplicate code

### PR DESCRIPTION
**Pull request by technicallyty**
## Description

- consolidates the Gogo/Pulsar compatibility code into one single types/ormutil directory, removing the duplicates found in core and basket.

Closes: n/a

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/data/server/query_server.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package server
 
 import (
 	"context"
+	"github.com/regen-network/regen-ledger/types/ormutil"
```
Comment (ryanchristo) [inline x/data/server/query_server.go:5]: Housekeeping

**Review by ryanchristo — APPROVED** [review]
Looks good to me. Would be nice to clean up import order.

**Comment by technicallyty** [comment]
> Looks good to me. Would be nice to clean up import order.

is there a plugin to group them? its a bit monotonous to reorder them by hand each time 😭

**Comment by ryanchristo** [comment]
> is there a plugin to auto-group them? its a bit monotonous to reorder them by hand each time sob

yea... I was hesitant to say anything but looks like you found something! :taco:

**Review by robert-zaremba — APPROVED** [review]

**Comment by robert-zaremba** [comment]
> edit2: doesnt appear to work correctly cry

It works, but it has different API, so you need to run it manually

**Review by aleem1314 — APPROVED** [review]

**Pull request merged**